## Additional_Assignment_16.1

### Problem Statement 1:

![PS_1.png](PS_1.png)

### Solution:

H0 : μ1 – μ2 = 0

H1 : μ1 – μ2 ≠ 0

![Z_Formula.png](Z_Formula.png)

In [1]:
import math

In [2]:
Z = ((452 - 523) - 0)/math.sqrt((math.pow(212,2)/1200) + (math.pow(185,2)/800))
Z

-7.926428526759299

p-value: p(z < -7.926) = 0

So this concludes that the null hypothesis (H0) can be rejected. The number of people travelling from Bangalore to Chennai is not same as the people travelling from Bangalore to Hosur.

### Problem Statement 2:

Is there evidence to conclude that the number of people preferring Duracell battery is
different from the number of people preferring Energizer battery, given the following:

Population 1: Duracell

n1 = 100

x1 = 308

s1 = 84

Population 2: Energizer

n2 = 100

x2 = 254

s2 = 67

### Solution:

H0 : μ1 – μ2 <= 45

H1 : μ1 – μ2 > 45

![Z_Formula.png](Z_Formula.png)

In [3]:
Z = ((308 - 254) - 45)/math.sqrt((math.pow(84,2)/100) + (math.pow(67,2)/100))
Z

0.837617111389407

p-value: p(z > 0.8376) = 1 - p(z < 0.84) = 1 - 0.7995 = 0.2005

p-value is less than z value. So we null hypothesis (H0) will be rejected at any level of significance.

### Problem Statement 3:

![PS_3.png](PS_3.png)

### Solution:

H0 : μ1 – μ2 = 0

H1 : μ1 – μ2 ≠ 0

![t_Formula.png](t_Formula.png)

In [4]:
t = ((0.317 - 0.21) - 0)/math.sqrt(((((14-1)*math.pow(0.12,2))+((9-1)*math.pow(0.11,2)))/(14+9-2))*((1/14)+(1/9)))
t

2.15355322387416

Critical points of t for 2 tail tests.

t(0.05) = 2.08, t(0.10) = 1.721

So we can reject null hypothesis (H0) at the 10% level of significance.

## Problem Statement 4:

The manufacturers of compact disk players want to test whether a small price reduction
is enough to increase sales of their product. Is there evidence that the small price
reduction is enough to increase sales of compact disk players?

Population 1: Before reduction

n1 = 15

x1 = Rs. 6598

s1 = Rs. 844

Population 2: After reduction

n2 = 12

x2 = RS. 6870

s2 = Rs. 669

### Solution:

H0 : μ1 – μ2 <= 0

H1 : μ1 – μ2 > 0

![t_Formula.png](t_Formula.png)

In [5]:
t = ((6598 - 6870) - 0)/math.sqrt(((((15-1)*math.pow(844,2))+((12-1)*math.pow(669,2)))/(15+12-2))*((1/15)+(1/12)))
t

-0.9098300343990461

alpha = 10% = 0.10

t critical value at 0.10 is ±1.316

So null hypothesis (H0) will not be rejected at the 10% level of significance.

## Problem Statement 5:

Comparisons of two population proportions when the hypothesized difference is zero.

Carry out a two-tailed test of the equality of banks’ share of the car loan market in 1980
and 1995.

Population 1: 1980

n1 = 1000

x1 = 53

𝑝̂ 1 = 0.53

Population 2: 1985

n2 = 100

x2 = 43

𝑝̂ 2= 0.53

### Solution:

H0 : p1 – p2 = 0

H1 : p1 – p2 ≠ 0

![Z_Formula_Probability.png](Z_Formula_Probability.png)

In [6]:
P = (53 + 43)/(1000 + 100)
P

0.08727272727272728

In [7]:
Z = ((0.53 - 0.43) - 0)/math.sqrt(P * (1-P) * ((1/1000) + (1/100)))
Z

3.378266212559581

Z critical value at 95% level of significance = 1.645

Z critical value at 99% level of significance = 2.33

So this concludes, the null hypothesis (H0) can be rejected even at 99% level of significance.

## Problem Statement 6:

Carry out a one-tailed test to determine whether the population proportion of traveler’s
check buyers who buy at least $2500 in checks when sweepstakes prizes are offered as
at least 10% higher than the proportion of such buyers when no sweepstakes are on.

Population 1: With sweepstakes

n1 = 300

x1 = 120

𝑝̂ 1 = 0.40

Population 2: No sweepstakes

n2 = 700

x2 = 140

𝑝̂ 2= 0.20

### Solution:

H0 : p1 – p2 < 0.10

H1 : p1 – p2 ≥ 0.10

![Z_Formula_Probability_1.png](Z_Formula_Probability_1.png)

In [8]:
Z = ((0.40 - 0.20) - 0.10)/math.sqrt(((0.4 * (1-0.4))/300) + ((0.2 * (1-0.2))/700))
Z

3.1180478223116177

alpha = 0.001

Z critical value at 0.001 level of significance is 3.09. So null hypothesis (H0) can be rejected at any level of significance.

## Problem Statement 7:

A die is thrown 132 times with the following results:

Number turned up: 1, 2, 3, 4, 5, 6

Frequency: 16, 20, 25, 14, 29, 28

Is the die unbiased? Consider the degrees of freedom as n − 1.

### Solution:

H0: Die is unbaised.

H1: Die is baised.

![PS_7_Solution.png](PS_7_Solution.png)

df = n - 1 = 6 - 1 = 5

alpha = 5% = 0.05

The critical value for χ2 at 0.05 is 11.07

As χ2 < χ2 (Critical), null hypothesis (H0) should not be rejected. We failed to reject H0.

## Problem Statement 8:

In a certain town, there are about one million eligible voters. A simple random sample of
10,000 eligible voters was chosen to study the relationship between gender and
participation in the last election. The results are summarized in the following 2X2 (read
two by two) contingency table:

![PS_8.png](PS_8.png)

We would want to check whether being a man or a woman (columns) is independent of
having
voted in the last election (rows). In other words, is “gender and voting independent”?

### Solution:

H0: Sex is independent of voting.

H1: Sex and voting are dependent.

![PS_8_Solution_1.png](PS_8_Solution_1.png)

![PS_8_Solution_2.png](PS_8_Solution_2.png)

df = (rows - 1) * (cols - 1) = (2-1) * (2-1) = 1

The critical value of χ2 at 5% level of significance when df = 1 is 3.84.

χ2 > χ2 (critical) => 6.584 > 3.84

So we reject the null hypothesis (H0).

## Problem Statement 9:

A sample of 100 voters are asked which of four candidates they would vote for in an
election. The number supporting each candidate is given below:

Higgins Reardon White Charlton

41      19      24      16

Do the data suggest that all candidates are equally popular? [Chi-Square = 14.96, with
3 df, 𝑝̂ < 0.05].

### Solution:

H0: All candidates are equally popular.

H1: All candidates are not equally popular.

![PS_9_Solution_1.png](PS_9_Solution_1.png)

df = n - 1 = 4 - 1 = 3

The critical value of Chi-Square for a 0.05 significance level and 3 d.f. is 7.82.

χ2 > χ2 (critical) => 14.96 > 7.82

So we reject null hypothesis (H0)

## Problem Statement 10:

Children of three ages are asked to indicate their preference for three photographs of
adults. Do the data suggest that there is a significant relationship between age and
photograph preference? What is wrong with this study? [Chi-Square = 29.6, with 4 df: 𝑝̂ <
0.05].

![PS_10.png](PS_10.png)

### Solution:

H0: There is no significance relationship between age and photography.

H1: There is a significance relationship between age and photography.

![PS_10_Solution.png](PS_10_Solution.png)

df = (rows - 1) * (cols - 1) = (3-1) * (3-1) = 2 * 2 = 4

The critical value of χ2 at 1% level of significance when df = 4 is 13.28.

χ2 > χ2 (critical) => 29.6032 > 13.28

So we reject the null hypothesis (H0).

## Problem Statement 11:

A study of conformity using the Asch paradigm involved two conditions: one where one
confederate supported the true judgement, and another where no confederate gave the
correct response.

![PS_11.png](PS_11.png)

### Solution:

H0: There are no significant difference between the "support" and "no support" conditions

H0: There are significant difference between the "support" and "no support" conditions

![PS_11_Solution.png](PS_11_Solution.png)

df = (rows - 1) * (cols - 1) = (2-1) * (2-1) = 1 * 1 = 1

The critical value of χ2 at 1% level of significance when df = 1 is 6.63.

χ2 > χ2 (critical) => 18.10345 > 6.63

So we reject the null hypothesis (H0).

## Problem Statement 12:

![PS_12.png](PS_12.png)

### Solution:

H0: There is no relationship between height and leadership qualities.

H1: There is a relationship between height and leadership qualities.

![PS_12_Solution.png](PS_12_Solution.png)

df = (rows - 1) * (cols - 1) = (3-1) * (2-1) = 2 * 1 = 2

The critical value of χ2 at 1% level of significance when df = 2 is 9.21

χ2 > χ2 (critical) => 10.9881 > 9.21

So we reject the null hypothesis (H0).

## Problem Statement 13:

Each respondent in the Current Population Survey of March 1993 was classified as
employed, unemployed, or outside the labor force. The results for men in California age
35-44 can be cross-tabulated by marital status, as follows:

![PS_13.png](PS_13.png)

Men of different marital status seem to have different distributions of labor force status.
Or is this just chance variation? (you may assume the table results from a simple random
sample.)

### Solution:

H0: There is no relationship between marital status and labour status.

H1: There is a relationship between marital status and labour status.

![PS_13_Solution.png](PS_13_Solution.png)

df = (rows - 1) * (cols - 1) = (3-1) * (3-1) = 2 * 2 = 4

The critical value of χ2 at 1% level of significance when df = 4 is 13.28

χ2 > χ2 (critical) => 30.9587 > 13.28

So we reject the null hypothesis (H0).